kubernetes_deploy/staging/deployment.yml

```diff
@@ -55,3 +56,62 @@ spec:
             secretKeyRef:
               name: db
               key: port
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-queue-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: laa-legal-adviser-api-queue-worker
+        env: staging
+        service_area: laa-get-access
+        service_team: cla-fala
+    spec:
+      containers:
+      - image: "<to be set by deploy_to_kubernetes>"
+        name: app
+        args: ["/usr/local/bin/python", "/home/app/manage.py", "celery", "worker", "-A laalaa", "--concurrency=1", "--loglevel=INFO"]
```

Comment:
What do you think about using `args: ["bin/run_worker"]`, with `bin/run_worker` being:
```sh
#!/bin/sh -e
./manage.py celery worker -A laalaa
```

Dissecting the changes:

* `manage.py` is an executable and there's only one python runtime in the image, so I feel we can skip `python`
* If the working directory of the app is `/home/app`, then we can skip that part (and it would make the script work locally too)
* Can we use configuration to default the concurrency rather than the command line?
* There is already a `LOG_LEVEL` environment variable below, it seems weird to pass it to `celery` again, could we use the environment variable?